docker-compose.yml

```diff
@@ -1,11 +1,31 @@
 version: '2'
 services:
   app:
+    environment:
+      - DB_URI=mongodb://mongo/oam-api
     env_file: .env
     build: .
+    depends_on:
+      - mongo
+    links:
+      - mongo
     image: oam-api
     restart: always
     ports:
       - 4000:4000
-
-
+    volumes:
+      - ./bin:/app/bin
```

Comment:
Is this for mounting development code? If so that should be put in the `test/docker-compose.yml` file, in fact, mounting live code is already solved in that file.

Reply:
Ditto for a local development environment. Each sub-directory / file is called out so that `node_modules` doesn't get mounted in (for cross-platform binary reasons).

I can keep this uncommitted locally.